src/applications/static-pages/analytics/addJumpLinkListeners.js

```diff
@@ -0,0 +1,23 @@
+import recordEvent from 'platform/monitoring/record-event';
+import { isAnchor } from './utilities';
+
+export default function addJumplinkListeners() {
+  const selectors = {
+    heading: '[data-template="paragraphs/wysiwyg"] #on-this-page',
```

Comment:
Is #on-this-page from debugging or is this an actual ID?

Reply:
An actual ID typed in the WYSIWYG but now I realize that ID was generated during the build by the heading's text

Reply:
Should it only record for this one jump link, or for all jump links?

If you want to make this more generic, could you use the [prefix attribute selector `^=`](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)?
```
'[data-template="paragraphs/wysiwyg"] [href^="#"]'
```

Reply:
This is actually a pretty fail solution because the problem is not good. We're only interested in the "table of contents" content model, but there's not one available to all layouts. Instead, this one is typed into a WYSIWYG and the only unique indicator of that is the heading ID, `on-this-page`. So we're actually interested in the click events on  sibling `<a>` elements, but use the heading ID, traverse to the parent element, then attach listeners for those click events